### PR TITLE
ceph-conf: fix typo in usage: 'mon add' should be 'mon addr'

### PR DESCRIPTION
**Pull request by zphj1987**
Find out what the value of 'mon addr' is for monitor 0.

Signed-off-by: Peng Zhang <zphj1987@gmail.com>

### CODE REVIEW

File: src/test/cli/ceph-conf/help.t [hunk 1]
```diff
@@ -26,7 +26,7 @@
   
   EXAMPLES
   [$] ceph-conf --name mon.0 -c /etc/ceph/ceph.conf 'mon addr' (re)
-  Find out what the value of 'mon add' is for monitor 0.
+  Find out what the value of 'mon addr' is for monitor 0.
```
Comment (tchaikov) [inline src/test/cli/ceph-conf/help.t]: nice catch. but you need to update `src/tools/ceph_conf.cc` accordingly.

Comment (zphj1987) [inline src/test/cli/ceph-conf/help.t]: i commit it ,is that right? thanks for your review  @tchaikov

Comment (tchaikov) [inline src/test/cli/ceph-conf/help.t]: could you prefix the title of your commit message with the subcomponent your are changing ? see https://github.com/ceph/ceph/blob/master/SubmittingPatches.rst#3-describe-your-changes

also, please fold your commits into a single one.

Comment (zphj1987) [inline src/test/cli/ceph-conf/help.t]: may be have wrong operation with more commit,i will resolve it tomorrow
Thank you for your help and patience

**Pull request closed without merging**